src/AstArgument.h

```diff
@@ -413,6 +413,12 @@ class AstUserDefinedFunctor : public AstFunctor {
         return args.size();
     }
 
+    /** set argument */
+    void setArg(size_t idx, std::unique_ptr<AstArgument> arg) {
+        assert(idx >= 0 && idx < args.size() && "wrong argument");
```

Comment:
"wrong argument" doesn't match the assertion. "invalid index for argument", perhaps?